mysql-test/suite/rpl/t/rpl_init_test_assert.test

```diff
@@ -0,0 +1,8 @@
+--source include/have_binlog_format_row.inc
```

Comment:
I don't think we have a convention yet for tests that validate mtr `.inc` files.. but I'm thinking we should, to differentiate that the test shouldn't be confused with a test of a server feature (e.g the variable `init_slave`, in this case). Perhaps changing the prefix from `rpl_` to `mtr_`? @andrelkin @knielsen any thoughts?